setfps.lua

```diff
@@ -15,6 +15,10 @@ Usage::
 local cap = ...
 local capnum = tonumber(cap)
 
+if not cap or cap=='-help' then
+    qerror('Usage:\n\tsetfps <number>')
```

Comment:
FYI: I think I'm going to change this to a `print()` call followed by a `return` (which exits the script without raising an error). That's a bit more consistent with other help text, which isn't usually printed as an error message (in red text).